v2.1/create-security-certificates-custom-ca.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Create Security Certificates
+summary: A secure CockroachDB cluster uses TLS for encrypted inter-node and client-node communication.
+toc: true
+---
+
+A secure CockroachDB cluster uses [TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security) for encrypted inter-node and client-node communication and requires a Certificate Authority (CA) certificate as well as keys and certificates for nodes, clients, and (optionally) the Admin UI. To create these certificates and keys, use the `cockroach cert` [commands](cockroach-commands.html) with the appropriate subcommands and flags, use [`openssl` commands](https://wiki.openssl.org/index.php/), or use a custom CA (for example, a public CA or your organizational CA).
+
+<div class="filters filters-big clearfix">
+  <a href="create-security-certificates.html"><button style="width:28%" class="filter-button">Use cockroach cert</button>
+  <a href="create-security-certificates-openssl.html"><button style="width:28%" class="filter-button">Use Openssl</button></a>
+  <button style="width:28%" class="filter-button current">Use <strong>custom CA</strong></button></a>
+</div>
+
+This document discusses the following advanced use cases for using security certificates with CockroachDB:
+
+Approach | Use case description
+-------------|------------
+UI certificate and key | When you want to access the Admin UI for a secure cluster and avoid clicking through a warning message to get to the UI.
+Split-node certificate | When your organizational CA requires you to have separate certificates for when the node acts as a server and as a client.
+Split-CA certificates | When you have multiple CockroachDB clusters and need to restrict access to clients from accessing the other cluster
+
+## Accessing the Admin UI for a secure cluster
+
+On [accessing the Admin UI](admin-ui-access-and-navigate.html#access-the-admin-ui), your browser will consider the CockroachDB-created certificate invalid, so you’ll need to click through a warning message to get to the UI.
+
+For secure clusters, you can avoid getting the warning message by using a certificate issued by a public CA:
+
+1. Request a certificate from a public CA (for example, [Let's Encrypt](https://letsencrypt.org/)). The certificate must have the IP addresses and DNS names used to reach the Admin UI listed in the `Subject Alternative Names` field.
```

Comment:
This should be: "listed in the Subject's `Common Name` field, or `Subject Alternative Name` field" (note the singular on "subject alternative name", applicable here and the rest of the doc).